app/experimenter/experiments/tasks.py

```diff
@@ -176,9 +184,13 @@ def update_status(experiment):
                 old_status=old_status,
                 new_status=new_status,
             )
-            metrics.incr("update_experiment_status.updated")
+            metrics.incr("update_experiment_info.updated")
             logger.info("Finished updating Experiment: {}".format(experiment))
 
+        if experiment.status == Experiment.STATUS_LIVE:
+            experiment = Experiment.objects.get(id=experiment.id)
```

Comment:
I think this is kinda awkward, but not sure what to do otherwise to retrieve the updated start and end date

Reply:
The `start_date` and `end_date` properties are methods that already do a bunch of database lookups when they're invoked, do you need to reload the instance in memory?

https://github.com/mozilla/experimenter/blob/master/app/experimenter/experiments/models.py#L397-L412

Reply:
I mean I could always default the start date to "today" and end date to today+ duration (is that awkward too?). Without reloading, the existing instance would still have the pre existing start and end date prior to the status change.